assertj-android-cardview-v7/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
   buildToolsVersion rootProject.ext.buildToolsVersion
 
   defaultConfig {
-    minSdkVersion 'L'
+    minSdkVersion 21
```

Comment:
I changed this bc I was getting a `INSTALL_FAILED_OLDER_SDK: Requires development platform L but this is a release platform.` when `./gradlew install` was trying to install apks.  Thoughts?